### ISSUE TITLE
Xcode build error (Cannot declare conformance to 'NSObjectProtocol')

### ISSUE THREAD

**Issue by epoxa** [react-native-text-input-mask/react-native-text-input-mask#204]
I got the following errors when build the RN project:
- node_modules/react-native-text-input-mask/ios/TextInputMask.swift:68:7: Cannot declare conformance to 'NSObjectProtocol' in Swift; 'MaskedRCTBackedTextFieldDelegateAdapter' should inherit 'NSObject' instead
- node_modules/react-native-text-input-mask/ios/TextInputMask.swift:68:49: Cannot find type 'RCTBackedTextFieldDelegateAdapter' in scope
"react-native-text-input-mask": "^3.1.2"
Xcode version is 12.4 (12D4e)
Are any ideas what can be wrong with it? Or what extra information should I provide?

**Comment by netmaxt3r** [comment]
If you have 
```ruby
pod 'React-RCTText', :path => '../node_modules/react-native/Libraries/Text', :modular_headers => true
```
in your podfile try adding 
```Swift
import RCTText
```
at the top of node_modules/react-native-text-input-mask/ios/TextInputMask.swift
if you are using patch-package use the below patch
```Diff
diff --git a/node_modules/react-native-text-input-mask/ios/TextInputMask.swift b/node_modules/react-native-text-input-mask/ios/TextInputMask.swift
index 7626c50..664a564 100644
--- a/node_modules/react-native-text-input-mask/ios/TextInputMask.swift
+++ b/node_modules/react-native-text-input-mask/ios/TextInputMask.swift
@@ -1,5 +1,6 @@
 import Foundation
 import InputMask
+import RCTText
 
 @objc(RNTextInputMask)
 class TextInputMask: NSObject, RCTBridgeModule, MaskedTextFieldDelegateListener {

```

**Comment by epoxa** [comment]
Thanks @netmaxt3r it compiles now without errors.
But I wonder why the import line is not present in original source code?

**Comment by netmaxt3r** [comment]
my guess is we both are compiling pods as  static lib without
```ruby
use_frameworks!
```

**Comment by martintreurnicht** [comment]
Yes this is correct, `use_frameworks!` is required as per the installation instructions. I've seen others suggest using modular headers on other issues, I'm happy to support this as an option if you want to submit a PR, as long as it doesn't cause any problems with `use_frameworks!`

**Comment by Nikolay-Vovk-dataart** [comment]
> Yes this is correct, `use_frameworks!` is required as per the installation instructions. I've seen others suggest using modular headers on other issues, I'm happy to support this as an option if you want to submit a PR, as long as it doesn't cause any problems with `use_frameworks!`

Hi. Is it possible to use the lib without `use_frameworks!`?  If  I use it it crashes pod installation because of other static deps

**Comment by netmaxt3r** [comment]
@martintreurnicht  please review PR #207

**Comment by martintreurnicht** [comment]
@netmaxt3r Thanks for the PR, merged and published as [3.1.4](https://github.com/react-native-text-input-mask/react-native-text-input-mask/releases/tag/v3.1.4)
